internal/dao/tests/account_test.go

```diff
@@ -108,7 +108,7 @@ func TestGetByOrgId(t *testing.T) {
 	assert.Equal(t, "1", account.AccountNumber.String)
 }
 
-func TestGetOrCreateByIdentityGet(t *testing.T) {
+func TestGetOrCreateByIdentityGetAccount(t *testing.T) {
 	accDao, ctx := setupAccount(t)
 	defer teardownAccount(t)
 	account, err := accDao.GetOrCreateByIdentity(ctx, "1", "1")
```

Comment:
Renamed these so I can test against "Account" pattern.

Reply:
What is "Account" pattern? :thinking:

Reply:
You can run tests against a regexp patter, so I was running all tests starting with Account, then with Pubkey etc. This allowed me to refactor and test DAO after DAO.